frontend/packages/console-plugin-sdk/src/useExtensions.ts

```diff
@@ -63,4 +63,4 @@ export const useExtensions: UseExtensions = (typeGuard) => {
   return extensionsInUse;
 };
 
-type UseExtensions = <E extends Extension>(typeGuard: ExtensionTypeGuard<E>) => readonly E[];
+type UseExtensions = <E extends Extension>(typeGuard: ExtensionTypeGuard<E>) => E[];
```

Comment:
FYI this change is to avoid TS errors like

```
The type 'readonly DashboardsTab[]' is 'readonly' and cannot be assigned to the mutable type 'DashboardsTab[]'.
```

since the `readonly` modifier actually yields a new type.

This isn't a big deal, though, since Console extension objects are frozen & immutable after their processing in `PluginStore`.